### PR TITLE
Smar 133

### PR DESCRIPTION
**Pull request by yulyaolshanska**
## Description changes:

- Weekly calendar in patient profile

## Issue ticket code (and/or) and link
[Link to JIRA ticket](https://smartdc23.atlassian.net/browse/SMAR-133)

### General
 
- [x] Assigned myself to the PR
- [x] Assigned the appropriate labels to the PR
- [x] Assigned the appropriate reviewers to the PR
- [ ] Updated the documentation
- [x] Performed a self-review of my code
- [x] Don't have "any" on my code
- [x] Don't have magic numbers
- [x] Compare only with constants not with strings
- [x] No ternary operator inside ternary operator
- [x] Don't have commented code
- [x] All links in on ENV files
- [x] Used camelCase for variables and functions
- [x] Functions is public only if it's used outside the class
- [x] No hardcoded values
- [x] Check your commit messages meet the [conventional commit format](https://www.conventionalcommits.org/en/v1.0.0/).

### Frontend
- [x] Components and business logic are separated
- [x] Colors, Font Size, and Font Name is on the theme or in the constants
- [x] No text in the components, use i18n approach
- [x] No inline styles
- [x] Imports are absolute
- [x] Attach a screenshot if PR has visual changes.

#### General view:

image

### CODE REVIEW

File: src/components/CardWrapper/index.tsx [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+import {
+  AccordionBody,
+  Arrow,
+  Container,
+  EditCardLink,
+  PatientName,
+} from './styles';
+
+type Props = {
```
Comment (DaniilVolobuiev) [inline src/components/CardWrapper/index.tsx]: use interface instead of type

File: src/components/CardWrapper/styles.ts [hunk 2]
```diff
@@ -0,0 +1,44 @@
+import { ViewLink } from '@components/PatientItem/styles';
+import { BLACK, BORDER, ZAMBEZI } from '@constants/colors';
+import { SMALL_FONT_SIZE } from '@constants/fontSizes';
+import styled from 'styled-components';
```
Comment (DaniilVolobuiev) [inline src/components/CardWrapper/styles.ts]: you should put node imports above the absolute

File: src/components/PatientList/index.tsx [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+  searchValue: string;
+}
+
+function PatientList({ searchValue }: IProps) {
+  const { t } = useTranslation();
+  const { data: patients } = useGetPatientsQuery('');
+  const [currentPage] = useState<number>(1);
+  const [displayedTweets, setDisplayedTweets] =
+    useState<number>(TWEETS_PER_PAGE);
+  const startIndex = (currentPage - 1) * TWEETS_PER_PAGE;
```
Comment (DaniilVolobuiev) [inline src/components/PatientList/index.tsx:19]: filter and pagination should be on backend. It is not rational to send a lot of patients to your client from the server. Instead you should send search params to server. But it of course depends on your backend)

File: src/components/CardWrapper/index.tsx [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+  const { t } = useTranslation();
+
+  const [toggle, setToggle] = React.useState<boolean>(false);
+  const [heightEl, setHeightEl] = React.useState<string>();
+
+  const refHeight = React.useRef<HTMLDivElement>(null);
+  const doctorData = useAppSelector((state) => state.doctorReducer);
+
+  const MARGIN_TOP = 20;
+  const PADDING = 16;
```
Comment (lyudad) [inline src/components/CardWrapper/index.tsx]: move out of component

File: src/utils/functions/getGoogleOauthURL.ts [hunk 5]
```diff
@@ -1,7 +1,7 @@
 function getGoogleOauthURL(): string {
   const rootUrl = 'https://accounts.google.com/o/oauth2/v2/auth';
   const options = {
-    redirect_uri: 'http://localhost:5000/auth/google/redirect',
+    redirect_uri: 'http://localhost:5001/auth/google/redirect',
```
Comment (lyudad) [inline src/utils/functions/getGoogleOauthURL.ts]: it should be in env

**Pull request merged**